packages/apps/storybook/src/addon/register.js

```diff
@@ -76,15 +76,15 @@ addons.register("project/toolbar", () => {
             const result = await response.text();
             const projects = JSON.parse(
               result.substring(0, result.indexOf("],")) + "]}"
-            ).projects;
+            ).iTwins;
 
             if (projects.length === 0) {
```

Comment:
why not rename it to iTwins instead of projects since it causes confusion and I think its better to stick with iTwins since the new API is not just limited to projects

Reply:
Because at this point, given that I requested only the "Project" `subClass`, these are only projects, not all the iTwins that were requested. The `subClass` param is required so I had to choose one, I went with the previous behavior.